test/functional/p2p_addrfetch.py

```diff
@@ -21,18 +21,24 @@
 
 
 class P2PAddrFetch(BitcoinTestFramework):
-
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
 
+    def assert_getpeerinfo(self, *, peer_ids):
```

Comment:
why did you opt for having peer_ids be an array when no call sites pass in >1 id?

Reply:
Right, the idea was that if a caller wants to pass more than one peer id, the helper function and all the existing callers don't need to be refactored.

Reply:
hm, since this is a helper within a specific test, this feels like a premature optimization to me- I would prefer simplifying the logic & increasing readability of the test.

Reply:
If it decreases the cost of future change at essentially no cost, it's good design. Leaving as it is and resolving.